serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/controller/ServiceRecordController.java

```diff
@@ -99,6 +105,20 @@ public ResponseEntity entity(
         return new ResponseEntity<>(organizationRes, HttpStatus.OK);
     }
 
+    @RequestMapping(value = "/identifier/{identifier}", method = RequestMethod.GET, produces = "application/jose")
+    @ResponseBody
+    public ResponseEntity signed(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: com.fasterxml.jackson.core.JsonProcessingException, java.security.cert.CertificateEncodingException, com.nimbusds.jose.JOSEException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1160)